core/domain/story_domain_test.py

```diff
@@ -495,6 +501,20 @@ def test_get_acquired_skill_ids_for_node_ids_overlapping_skills(
             ['skill_1', 'skill_2']
         )
 
+    def test_get_acquired_skill_ids_for_node_ids_not_found(
+        self
+    ) -> None:
+        # Test cases when node.id not in node_ids.
```

Comment:
Comments like this shouldn't be needed if you've named the test well

Reply:
done